lib/sidekiq/cron/job.rb

```diff
@@ -233,7 +233,7 @@ def self.find name
             output = Job.new conn.hgetall( redis_key(name) )
           end
         end
-        output
+        output if output&.valid?
```

Comment:
Needs ruby 2.2 support.

```suggestion
        output if output && output.valid?
```